notebooks/277-amused-lightweight-text-to-image/277-amused-lightweight-text-to-image.ipynb

```diff
@@ -0,0 +1,667 @@
+{
```

Comment:
<p>in review window this image looks very large and low quality. Can we reduce its size or remove?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>